code/game/objects/structures/flora.dm

```diff
@@ -205,4 +201,149 @@
+	icon_state = "plant-10"
+
+/obj/structure/flora/pottedplant/smallcactus
+	name = "small potted cactus"
+	desc = "This is a small cactus. Its needles are sharp."
+	icon_state = "plant-11"
+
+/obj/structure/flora/pottedplant/tall
+	name = "tall potted plant"
+	desc = "This is a tall plant. Tiny pores line its surface."
+	icon_state = "plant-12"
+
+/obj/structure/flora/pottedplant/sticky
+	name = "styicky potted plant"
+	desc = "This is an odd plant. Its sticky leaves trap insects."
+	icon_state = "plant-13"
+
+/obj/structure/flora/pottedplant/smelly
+	name = "smelly potted plant"
+	desc = "This is some kind of tropical plant. It reeks of rotten eggs."
+	icon_state = "plant-14"
+
+/obj/structure/flora/pottedplant/small
+	name = "small potted plant"
+	desc = "This is a pot of assorted small flora. Some look familiar."
+	icon_state = "plant-15"
+
+/obj/structure/flora/pottedplant/aquatic
+	name = "aquatic potted plant"
+	desc = "This is apparently an aquatic plant. It's probably fake."
+	icon_state = "plant-16"
+
+/obj/structure/flora/pottedplant/shoot
+	name = "small potted shoot"
+	desc = "This is a small shoot. It still needs time to grow."
+	icon_state = "plant-17"
+
+/obj/structure/flora/pottedplant/flower
+	name = "potted flower"
+	desc = "This is a slim plant. Sweet smelling flowers are supported by spindly stems."
+	icon_state = "plant-18"
+
+/obj/structure/flora/pottedplant/crystal
+	name = "crystalline potted plant"
+	desc = "These are rather cubic plants. Odd crystal formations grow on the end."
+	icon_state = "plant-19"
+
+/obj/structure/flora/pottedplant/subterranean
+	name = "subterranean potted plant"
+	desc = "This is a subterranean plant. It's bulbous ends glow faintly."
+	icon_state = "plant-20"
+	set_light(l_range = 1, l_power = 0.5, l_color = "#FF6633")
+
+/obj/structure/flora/pottedplant/minitree
+	name = "potted tree"
+	desc = "This is a miniature tree. Apparently it was grown to 1/5 scale."
+	icon_state = "plant-21"
+
+/obj/structure/flora/pottedplant/stoutbush
+	name = "stout potted bush"
+	desc = "This is a stout bush. Its leaves point up and outwards."
+	icon_state = "plant-22"
+
+/obj/structure/flora/pottedplant/drooping
+	name = "drooping potted plant"
+	desc = "This is a small plant. The drooping leaves make it look like its wilted."
+	icon_state = "plant-23"
+
+/obj/structure/flora/pottedplant/tropical
+	name = "tropical potted plant"
+	desc = "This is some kind of tropical plant. It hasn't begun to flower yet."
+	icon_state = "plant-24"
+
+/obj/structure/flora/pottedplant/dead
+	name = "dead potted plant"
+	desc = "This is the dried up remains of a dead plant. Someone should replace it."
+	icon_state = "plant-25"
+
+/obj/structure/flora/pottedplant/decorative
+	name = "decorative potted plant"
+	desc = "This is a decorative shrub. It's been trimmed into the shape of an apple."
+	icon_state = "applebush"
+
+/obj/structure/flora/pottedplant/xmas/New()

```

Comment:
Don't think this should be a New(). The New() you replaced basically only renamed the item, so it's unnecessary now, as far as I can tell.